src/Sylius/Component/Shipping/Model/ShipmentInterface.php

```diff
@@ -48,7 +48,7 @@ public function getMethod();
     /**
      * @param ShippingMethodInterface $method
      */
-    public function setMethod(ShippingMethodInterface $method);
+    public function setMethod(ShippingMethodInterface $method = null);
```

Comment:
Unfortunately we must allow setting ``null`` on shipment method to make validation works. It must get object with invalid state to add violation, otherwise selecting no shipping method in shipping step would lead to an exception.